TESTING.md

```diff
@@ -66,15 +66,37 @@ For manual testing, you can use the utility script [scripts/run-external.sh](scr
 ./scripts/run-external.sh -c
 ```
 
-Before running automated end-to-end tests, you need run the following command to make images and load it in your local cluster (for KinD cluster):
+### Building Images and Loading them into your cluster
+
+#### Using docker with Kind
+
+Before running automated end-to-end tests, you need run the following command to make images and load it in your local cluster:
 
 ```
 make image
 
 for n in "prometheus-operator" "prometheus-config-reloader" "admission-webhook"; do kind load docker-image "quay.io/prometheus-operator/$n:$(git rev-parse --short HEAD)"; done;
 ```
 
-Then to run the automated end-to-end tests, run the following command:
+#### Using podman with Kind
+
+When running kind on MacOS using podman, it is recommended to create podman machine 4 CPUs and 8 GiB memory. Less resources might cause end to end tests fail because of lack of resources for cluster.
+
+```
+podman machine init --cpus=4 --memory=8192 --rootful --now
+```
+
+Before running automated end-to-end tests, you need run the following command to make images and load it in your local cluster:
+
+```
+CONTAINER_CLI=podman make image
+
+for n in "prometheus-operator" "prometheus-config-reloader" "admission-webhook"; do podman save --quiet -o images/$n.tar "quay.io/prometheus-operator/$n:$(git rev-parse --short HEAD)"; kind load image-archive images/$n.tar; done
```

Comment:
There is no images directory either need to create it or lets use existing `tmp` that gets created anyway with make image

```suggestion
for n in "prometheus-operator" "prometheus-config-reloader" "admission-webhook"; do podman save --quiet -o tmp/$n.tar "quay.io/prometheus-operator/$n:$(git rev-parse --short HEAD)"; kind load image-archive tmp/$n.tar; done
```

Reply:
I was not aware of this one thank you for that.